GCMFormTableView/GCMFormTableView/GCMItemSelectTableViewDataSource.m

```diff
@@ -307,6 +307,29 @@ - (NSDictionary *)defaultHeaderFooterTextAttributes {
            NSParagraphStyleAttributeName : paragraphStyle};
 }
 
+- (void)handleSelectionAtIndexPath:(NSIndexPath *)indexPath onTableView:(UITableView *)tableView {
+  GCMItemSelectSection *itemSection = self.sections[indexPath.section];
+  GCMItemSelectItem *item = itemSection.items[indexPath.row];
+  if ( item.disabled ) {
+    return;
+  }
+  
+  if ( item.actionItem ) {
+    [self reportSelectedActionItemForIndexPath:indexPath];
+  } else {
+    if ( ! [self.selectedIndexPath isEqual:indexPath] ) {
```

Comment:
Let's only do this block if `tableView != nil` just to make some things more explicit.